### PR TITLE
Timezone bug in VZE fatalities view and fix engineering area label

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18930
Closes https://github.com/cityofaustin/atd-data-tech/issues/18955

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch, start your local db, apply up migrations and metadata. 
2. Go to the Fatalities page in the VZE, filter the dates to be between December 2019 and January 2020, sort by the crash date column.
3. See that the engineering area column now shoes the actual label of the engineering area and not the id
4. See that this [crash](https://visionzero.austin.gov/editor/#/crashes/17492001) (cris_crash_id: 17492001) which occurred on the last day of 2019 is no longer is showing that its the 1st fatality of the year in the ytd_fatality column, it should be 88.
5. Test running the down migrations

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vzd/migrations/default/1726000511912_fix_timezone_fatalities_view/up.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'YYYY-MM-DD'
+  ) as crash_date_ct,
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'HH24:MI:SS'
+  ) as crash_time_ct,
+-- get ytd fatality, partition by year and sort by date timestamp
+ROW_NUMBER() OVER (
+    PARTITION BY EXTRACT(year FROM crashes.crash_timestamp at time zone 'US/Central') 
+    ORDER BY crashes.crash_timestamp at time zone 'US/Central' ASC)
```
Comment (roseeichelmann) [inline atd-vzd/migrations/default/1726000511912_fix_timezone_fatalities_view/up.sql:29]: adding this `at time zone 'US/Central'` to these ytd_fatality and ytd_fatal_crash fields is what fixed the bug

File: atd-vzd/metadata/databases/default/tables/public_crashes.yaml [hunk 2]
```diff
@@ -92,7 +92,7 @@ select_permissions:
         - crash_timestamp
         - created_at
         - created_by
-        - engineering_area
+        - engineering_area_id
```
Comment (roseeichelmann) [inline atd-vzd/metadata/databases/default/tables/public_crashes.yaml:95]: decided to rename this column to `engineering_area_id` to differentiate between id and the actual name of the eng area

File: atd-vzd/migrations/default/1726084839739_update_crashes_set_spatial_attributes/up.sql [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            from
+                public.council_districts
+            where
+                st_contains(geometry, new.position)
+            limit 1);
+        raise debug 'council_district: % compared to previous: %', new.council_district, old.council_district;
+        --
+        -- get engineering area
+        --
+        new.engineering_area_id = (
```
Comment (roseeichelmann) [inline atd-vzd/migrations/default/1726084839739_update_crashes_set_spatial_attributes/up.sql:58]: updated this section to use the new `engineering_area_id` column name

File: atd-vze/src/views/Fatalities/fatalityGridTableParameters.js [hunk 4]
```diff
@@ -70,7 +70,7 @@ export const fatalityGridTableColumns = {
     label_table: "FRB Recommendation",
     type: "String",
   },
-  engineering_area: {
+  "engineering_area { label }": {
```
Comment (roseeichelmann) [inline atd-vze/src/views/Fatalities/fatalityGridTableParameters.js:73]: query relationship to `engineering_area` table to get the label instead of id

**Review by johnclary — APPROVED** [review]
everything is working great—i'm really glad you renamed this column. thanks for this PR! 🚢 🚢 🚢 🚢 🚢 🚢 🚢

**Comment by roseeichelmann** [comment]
@chiaberry thanks for catching that. i fixed the migrations by moving them all to one folder/one file so that i could control the order of all the commands :pray:

**Review by chiaberry — APPROVED** [review]
up and down migrations ✅ 
time is still a construct!

**Pull request merged**